src/files.js

```diff
@@ -491,7 +491,6 @@ FOAM_FILES([
   { name: "foam/swift/refines/Topic", flags: ['swift'] },
   { name: "foam/swift/refines/Remote", flags: ['swift'] },
 
-  { name: "foam/nanos/menu/DAOMenu2" },
```

Comment:
How do things work if you remove this?

Reply:
its already in nanos.js

https://github.com/kgrgreer/foam3/blob/development/src/foam/nanos/nanos.js#L167